### DISCUSSION TITLE
How to increase the width of the coverage summary report in the terminal?

### DISCUSSION THREAD

**Discussion by ValentinGurkov** [vitest-dev/vitest#5270] [Q&A] [answered]
Hello everyone,

I've tried different terminals and my test result summary always looks this squished:
![image](https://github.com/vitest-dev/vitest/assets/35373940/885282bc-16fd-4e4e-8a8d-7ffe5929ad2a)

This happens even when there is a lot of free space on the screen and I can't find a setting to adjust this.

Probably if I parse a json report with `istanbul` I can adjust the look of the report table but I'd rather not go that far...

Does anybody know of a way?

**Comment by AriPerkkio** [comment, marked as answer]
The `text` reporter accepts these options: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/276d95e4304b3670eaf6e8e5a7ea9e265a14e338/types/istanbul-reports/index.d.ts#L62-L65

Maybe this works:

```ts
{
  coverage: {
    reporter: [
      ['text', { maxCols: 200 }],
    ]
  }
}
```

**Reply by ValentinGurkov** [reply]
Thank you so much!
